src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -14,18 +13,14 @@ public class AddDriverController extends HttpServlet {
     private final DriverService driverService = (DriverService) injector
             .getInstance(DriverService.class);
 
-    @Override
-    public void doGet(HttpServletRequest req, HttpServletResponse resp)
-            throws ServletException, IOException {
-        req.getRequestDispatcher("/WEB-INF/views/drivers/add.jsp").forward(req, resp);
-    }
```

Comment:
please not display drivers/add.jsp together with displaying all drivers
restore this code and let's use this jsp for registration

we should not be able to add new driver on `all drivers` page - we should now add new driver only via registration